src/flash/nor/mspm0.c: Update protection status output and removing e…

HenryNguyen01 opened the pull request:
…xcess functions

According to reviews, the implementation of outputting the protection status was not ideal. To address reviewers concern the status of protection will always display unprotected. Within this commit other protection functions are removed as well since they will be never utilized due to how protection is displayed.

Change-Id: I4111506a7ef5c45e483e34b4a6adaa855596daee

HNguyen00 committed:
src/flash/nor/mspm0.c: Update protection status output and removing excess functions

According to reviews, the implementation of outputting the protection status
was not ideal. To address reviewers concern the status of protection will
always display unprotected. Within this commit other protection functions
are removed as well since they will be never utilized due to how protection
is displayed.

Change-Id: I4111506a7ef5c45e483e34b4a6adaa855596daee
Signed-off-by: Henry Nguyen <h-nguyen8@ti.com>

nmenon commented:
We should add documentation on how the protection scheme is. People will expect that.

HNguyen00 committed:
src/flash/nor/mspm0.c: Adding comment to protection check function

Provided more context as to why the protection is handled in such
manner with the implementation.

Change-Id: I7e4088d5d8dfade270f6a00072e1ea7deb79cca7
Signed-off-by: Henry Nguyen <h-nguyen8@ti.com>

HenryNguyen01 commented:
> We should add documentation on how the protection scheme is. People will expect that.

Sounds good, and done
